roles/cloudera_manager/external_auth/tasks/main.yml

```diff
@@ -14,6 +14,14 @@
 
 ---
 
+- name: Initialize for IPA
+  include_role:
+    name: initialize_for_ipa
```

Comment:
Where is this role?

Reply:
dont understand the question, "where is ?"

Reply:
There is no role name `initialize_for_ipa`. There is a task list called this name, but not a role.

Reply:
done